cost/azure/reserved_instances/recommendations/README.md

```diff
@@ -23,26 +17,32 @@ This policy has the following input parameters required when launching the polic
 - *Net Savings Threshold* - Specify the minimum Net Savings that should result in an RI purchase recommendation
 - *Email addresses to notify* - A list of email addresses to notify
 
-### Policy Actions
+## Policy Actions
 
 The following policy actions are taken on any resources found to be out of compliance.
 
 - Send an email report
 
-### Required RightScale Roles
+## Prerequesites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html)
+for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no
+credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed: 
 
-- Cloud Management - Actor
-- Cloud Management - Observer
-- Cloud Management - credential_viewer
+Provider tag value to match this policy: `azure`
```

Comment:
Azure EA policies use the provider tag azure_ea

Reply:
Updated the provider tag.